CLR/Libraries/Windows_Devices/windows_devices_stub.cpp

```diff
@@ -0,0 +1,61 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#include "windows_devices.h"
+
+HRESULT Library_windows_devices_native_Windows_Devices_Gpio_GpioController::get_PinCount___I4( CLR_RT_StackFrame& stack )
+{
+    TINYCLR_HEADER();
+    TINYCLR_SET_AND_LEAVE(stack.NotImplementedStub());

```

Comment:
Change the exception return here (and all the others) to Not Supported, Not implemented is mostly a runtime form of TODO: comment, that shouldn't be left in production code. Also, not supported is semantically more correct with a stub library which is literally in place because the given functionality isn't supported in this firmware build.